buildSrc/src/main/java/org/opensearch/gradle/internal/InternalDistributionBwcSetupPlugin.java

```diff
@@ -181,15 +181,19 @@ private static List<DistributionProject> resolveArchiveProjects(File checkoutDir
                 if (name.contains("zip") || name.contains("tar")) {
                     int index = name.lastIndexOf('-');
                     String baseName = name.substring(0, index);
-                    classifier = "-" + baseName + "-x64";
+                    classifier = "-" + baseName;
+                    // The x64 variants do not have the architecture built into the task name, so it needs to be appended
+                    if (name.equals("darwin-tar") || name.equals("linux-tar") || name.equals("windows-zip")) {
+                        classifier += "-x64";
+                    }
                     extension = name.substring(index + 1);
                     if (extension.equals("tar")) {
                         extension += ".gz";
                     }
                 } else if (name.contains("deb")) {
-                    classifier = "-amd64";
+                    classifier = "_amd64";
                 } else if (name.contains("rpm")) {
-                    classifier = "-x64";
+                    classifier = ".x86_64";
```

Comment:
I think this change is not right, here is the `.deb` and `.rpm` for latest release:
`https://artifacts.opensearch.org/releases/bundle/opensearch/2.18.0/opensearch-2.18.0-linux-x64.deb`
`https://artifacts.opensearch.org/releases/bundle/opensearch/2.18.0/opensearch-2.18.0-linux-x64.rpm`

Reply:
If I understand things correctly, you're linking to the distribution target built by the scripts that live in opensearch-build. Here's what I get when running the gradle tasks in this repo:

```
$ ./gradlew distribution:bwc:minor:buildBwcRpm
...
$ ls /home/ubuntu/OpenSearch/distribution/bwc/minor/build/bwc/checkout-2.x/distribution/packages/rpm/build/distributions
opensearch-min-2.19.0-SNAPSHOT.x86_64.rpm
```

Reply:
Oh I see, correct, but shouldn't those be the same? Or there is a separate build for those? To be fair, `x86_64` looks correct to me, but difference is quite confusing

Reply:
I believe opensearch-build takes the RPM built by the min distribution, opens it up, adds all the other plugins, then packages it back into an RPM which is the artifact that is released. I suppose ideally the artifact names would be the same (except for the "-min" part) but I'm not sure it is worth the effort to change it.

Reply:
I see, thanks @andrross